routes/students.js

```diff
@@ -12,6 +12,51 @@ router.get('', async (req, res) => {
   }
 });
 
+router.post('/create', async(req, res) => {
+  try {
+    const student = req.body;
+    console.log(student);
+    const newStudent = await pool.query(
+      'INSERT INTO student (first_name, last_name, contact, site_id, student_group) VALUES ($1, $2, $3, $4, $5) RETURNING student_id',
+      [student.first_name, student.last_name, student.contact, student.site_id, student.student_group],
+    );
+    res.json(newStudent.rows[0]);
+  } catch (err) {
+    console.error(err.message);
+  }
+});
+
+router.post('/update-scores/:id', async(req, res) => {
+  try {
+    const student = req.body;
+    console.log(student);
+    const studentScores = await pool.query(
+      'INSERT INTO student (pretest_r, posttest_r, pretest_a, posttest_a) VALUES ($1, $2, $3, $4) RETURNING * ',
+      [student.pretest_r, student.posttest_r, student.pretest_a, student.posttest_a],
+    );
+    res.json("Student's scores was successfully updated.")
+  } catch (err) {
+    console.error(err.message);
+  }
+});
+
+router.put('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { first_name } = req.body;
+    const { last_name } = req.body;
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'last_name' is not in camel case.